### PR TITLE
[couch-to-sql][Repeaters][Phase-3] Remove couch repeater models and its references

### PR DESCRIPTION
**Pull request by AmitPhulera**
## Product Description

The PR removes all the migration code along with the couch models and their references. This PR does not refactor the SQL Model names as it will be a significantly low-effort change, I will do it in a follow-up PR after this is merged.

## Technical Summary

https://dimagi-dev.atlassian.net/browse/SAAS-12155
### Safety story

Changes have been tested locally and I would be putting up a small QA request for it. 
### QA Plan

Will link to the ticket soon

### Rollback instructions

- [x] This PR can be reverted after deploy with no further considerations

### Labels & Review
- [x] Risk label is set correctly
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change

### CODE REVIEW

**Review by millerdev — APPROVED** [review]
Looks great! Left two minor comments that are not crucial, but would be nice to have addressed.

File: corehq/motech/repeaters/models.py [hunk 1]
```diff
@@ -1517,144 +842,13 @@ def get_repeater_response_from_submission_response(response):
...
-        (
-            to_function(class_path, failhard=True).__name__,
-            to_function(class_path, failhard=True)._migration_get_sql_model_class()
+    return OrderedDict(
+        list(
+            zip(REPEATER_CLASS_MAP.keys(), REPEATER_CLASS_MAP.values())
         )
-        for class_path in settings.REPEATER_CLASSES
-    ])
+    )
```
[the remark this answers is not in the document]
Comment (AmitPhulera) [reply corehq/motech/repeaters/models.py:851]: Ohh! I didn't know that. Thanks for the suggestion :)

File: corehq/motech/repeaters/models.py [hunk 2]
```diff
@@ -263,7 +259,7 @@ def by_domain(self, domain):
         return list(self.filter(domain=domain))
 
 
-class SQLRepeater(SyncSQLToCouchMixin, RepeaterSuperProxy):
+class SQLRepeater(RepeaterSuperProxy):
```
[the remark this answers is not in the document]
Comment (AmitPhulera) [reply corehq/motech/repeaters/models.py:250]: https://github.com/dimagi/commcare-hq/pull/32470/commits/50d832a2d4ed626fae7ec697f48ddcd09005475a

File: corehq/motech/repeaters/models.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (AmitPhulera) [reply corehq/motech/repeaters/models.py]: https://github.com/dimagi/commcare-hq/pull/32470/commits/18fe7e972d4710a68407f8a8bd0757b442c4d4a3

**Review by millerdev — APPROVED** [review]

**Comment by dannyroberts** [comment]
Are there any couch views we can delete now because of this? (Sorry if I missed this elsewhere!)

**Comment by AmitPhulera** [comment]
@dannyroberts We can delete `repeaters/repeaters` view in `commcarehq__receiverwrapper`.

**Comment by dannyroberts** [comment]
I see, that's mixed in in the same design doc with the (much larger) repeat records views, so deleting `repeaters/repeaters` is probably not worth the big reindex it would cause. So I think it's fine to leave as is. Thanks!

**Pull request merged**